app/kube-knots/src/components/resource-table.tsx

```diff
@@ -43,13 +43,13 @@ export function ResourceTable<T extends ResourceBase>({
   actions,
   renderData,
 }: ResourceListProps<T>) {
-  const resourceListQuery = useResourceList<T>(command);
-
   const { selected, handleOpen, handleClose, action } = useResourceActions<
     T,
     (typeof actions)[number]
   >();
 
+  const resourceListQuery = useResourceList<T>(command, undefined, { enabled: !action });
```

Comment:
basically disable the query if we have an action, meaning there is some sort of overlay.